libs/core/src/lib/fixed-card-layout/fixed-card-layout-item/fixed-card-layout-item.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, ElementRef, ChangeDetectionStrategy, HostBinding } from '@angular/core';
+import { FocusableOption } from '@angular/cdk/a11y';
+
+@Component({
+    selector: 'fd-fixed-card-layout-item',
+    template: `<ng-content></ng-content>`,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        style: 'display: block'
+    }
+})
+export class FixedCardLayoutItemComponent implements FocusableOption {
+    @HostBinding()
```

Comment:
Let's add `/** @hidden */` comment, to hide it from docs

Reply:
added /** @hidden */ to hide from docs.